source/MaterialXContrib/MaterialXMaya/CreateMaterialXNodeCmd.cpp

```diff
@@ -220,36 +221,35 @@ MStatus CreateMaterialXNodeCmd::doIt( const MArgList &args )
             CHECK_MSTATUS(argData.getFlagArgument(kEnvIrradianceFlagLong, 0, envIrradianceFileName));
         }
 
-        const mx::FileSearchPath& searchPath = Plugin::instance().getLibrarySearchPath();
-        MStringArray nodeNames;
-        for (auto renderableElement : renderableElements)
+        MStringArray createdNodeNames;
+        for (const auto& renderableElement : renderableElements)
         {
             std::string nodeName = createNode(  document,
                                                 renderableElement,
-                                                createAsTexture,
+                                                createAs,
                                                 documentFilePath,
-                                                searchPath,
+                                                Plugin::instance().getLibrarySearchPath(),
                                                 envRadianceFileName,
                                                 envIrradianceFileName );
-            nodeNames.append(nodeName.c_str());
+            createdNodeNames.append(nodeName.c_str());
         }
 
         status = _dgModifier.doIt();
         if (!status)
         {
-            unsigned int nodeNameCount = nodeNames.length();
-            std::string nodeString;
-            if (nodeNameCount)
+            const unsigned int nCreatedNodes = createdNodeNames.length();
```

Comment:
Pickiness on naming. Can we not use "n"CreatedNodes as it's not English. Maybe just get rid of it as it's only used once at line 242.

Reply:
I think he's just trying to indicate that it's an int, but I don't think we tend to do this with our naming convention.

Reply:
`i`, `msg` or `e` that our codebase uses often are not English either :-) In my mind, `n` stands for "number of" here and it admittedly comes from much-maligned Hungarian notation. OK, I'll rename it to something more verbose.

Reply:
Renamed.

Reply:
We don't use Hungarian notation. Thanks for the rename though as said you could have just deleted the temporary :).

Reply:
It is used in two places though :-)